readlink: add page (#6104)
tldr-pages/tldr: pages/osx/readlink.md

# readlink

> Follow symlinks and get symlink information.
> More information: <https://www.gnu.org/software/coreutils/readlink>.

> - Get the actual file to which the symlink points:

Reviewer: ```suggestion
- Print the actual full path to which the symlink points:
```

> - Print the actual full path to which the symlink points:

Reviewer 2: ```suggestion
- Print the actual full path to which the symlink points to:
```
thoughts?

Author: sounds good but I;m no english expert..

Reviewer: `to` is already there ("... full path **to** which the ...")

Reviewer: No, it's redundant.  I personally prefer the original because it's more advanced-sounding :) but it's probably less readable for non-native-English-speakers.
```suggestion
- Print the actual full path which the symlink points to:
```

Author: so what sounds better
1) `Print the actual full path to which the symlink points:`
2) `Print the actual full path which the symlink points to:`

I think 2 sounds better

Reviewer 2: Yes, both works and the current version is more concise, however with 2 `to` it reads nicer and releases the formality of it which it doesn't really need.

Author: so should I change it to 2?

Reviewer: With two `to`s it's actually grammatically incorrect, or at least inappropriately redundant.

The two are both **100% equivalent**, the only difference is the position.  I like the first because I'm a native speaker so interesting twists appeal to me, but most people reading these pages won't feel the same way and will be confused.  Please, @Itai-Nelken, apply my latest [suggestion](https://github.com/tldr-pages/tldr/pull/6104#discussion_r646960844) above :)

Author: when I said 2 I meant my 2nd suggestion that is the same as yours but I missed it

> - Print the actual full path which the symlink points to:

Reviewer 3: ```suggestion
- Print the absolute path which the symlink points to:
```

Just my opinion but could we use the word absolute instead of full?
Paths are relative or absolute, while "full" is recognized/valid, it's less common.

https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths

I also feel the use of the word "actual" is redundant here. Usually, when the word "actual" or "actually" like this, it's to add emphasis, but that's not needed here.

Any other opinions?

Author: sounds better to me, I'll replace it.

> `readlink {{filename}}`

Reviewer: I'd maybe say `path/to/symlink` as well, wdyt?
```suggestion
`readlink {{path/to/symlink}}`
```

Author: good idea
